### DISCUSSION TITLE
Don't set default path to user Documents by default

### DISCUSSION THREAD

**Discussion by lalitshankarch** [godotengine/godot-proposals#10871] [Editor]
Currently Godot chooses the user Documents folder by default on Windows:

![image](https://github.com/user-attachments/assets/a1533a2c-f7a8-4fe1-9ee1-0a44eede3bf1)

Ideally it should set the project folder location to the User home by default.

**Comment by Calinou** [comment]
> Ideally it should set the project folder location to the User home by default.

This would encourage messy organization where lots of folders would be created in the user's home folder, so I don't think it's a good idea. There are already too many apps out there that create folders directly in the home directory :slightly_smiling_face: 

Personally, I'd go the opposite route and use `Documents/Godot` by default (with `Godot` being automatically created if it doesn't exist), but not everyone agrees: https://github.com/godotengine/godot/pull/47724

That said, if coupled with https://github.com/godotengine/godot-proposals/issues/1312, it could be a reasonable solution.
